Week-12/query.sql

```diff
@@ -0,0 +1,23 @@
+-- Problem1 
+SELECT min(WEIGHT) AS WEIGHT FROM ITEMS;
```

Comment:
Here it will only return the min weight but not the information of the item with this weight.